exercises/sublist/.meta/HINTS.md

```diff
@@ -0,0 +1,10 @@
+This exercise stub and the test suite contain several enumerated constants. 
+
+Since Python 2 does not have the enum module, the idiomatic way to write 
+enumerated constants has traditionally been a NAME assigned to an arbitrary, 
+but unique value. An integer is traditionally used because it’s memory 
+efficient.
+It is a common practice to export both constants and functions that work with 
+those constants (ex. the constants in the os, subprocess and re modules). 
+
+You can learn more here: https://en.wikipedia.org/wiki/Enumerated_type
```

Comment:
Feel free to suggest edits to this file.

Reply:
I'd have to look at bin/check-readmes.sh to see why that error would occur, but that said I'd propose deleting this file and instead including the hints above as comments at the top of the `sublist.py` file.

Reply:
The error occurred because the README.md for the exercise was not re-generated to include this new content.